cfgrammar/src/lib/header.rs

```diff
@@ -144,21 +143,26 @@ impl<'input> GrmtoolsSectionParser<'input> {
         }
     }
 
-    fn parse_path(&'_ self, mut i: usize) -> Result<(Path<'_>, usize), HeaderError> {
+    fn parse_namespaced(&self, mut i: usize) -> Result<(Namespaced, usize), HeaderError> {
+        // Either a name without a namespace, or a namespace.
         let (name, j) = self.parse_name(i)?;
         let name_span = Span::new(i, j);
         i = self.parse_ws(j);
         if let Some(j) = self.lookahead_is("::", i) {
             i = self.parse_ws(j);
-            let (scoped_val, j) = self.parse_name(i)?;
-            let scoped_span = Span::new(i, j);
+            let (namespaced_val, j) = self.parse_name(i)?;
+            let namespaced_val_span = Span::new(i, j);
             i = self.parse_ws(j);
             Ok((
-                Path::Scoped((name, name_span), (scoped_val, scoped_span)),
+                (
+                    Some((name, name_span)),
+                    (namespaced_val, namespaced_val_span),
+                ),
                 i,
             ))
         } else {
-            Ok((Path::Ident(name, name_span), i))
+            // Without a namespace.
```

Comment:
Comment seems superfluous now that we've made this a struct.

Reply:
Should be fixed in [1fefed0](https://github.com/softdevteam/grmtools/pull/546/commits/1fefed0770fc22f6511a8848a52330b921feec89)

Also the locals here could be more clear, being more similar to the struct field names.